ztp/gitops-subscriptions/argocd/example/policygentemplates/group-du-sno-and-3node-validator-ranGen.yaml

```diff
@@ -0,0 +1,16 @@
+---
+apiVersion: ran.openshift.io/v1
+kind: PolicyGenTemplate
+metadata:
+  # The name will be used to generate the placementBinding and placementRule names as {name}-placementBinding and {name}-placementRule
+  name: "group-du-sno-and-3node-validator"
```

Comment:
If we want a less cumbersome name, maybe "group-du-master-validator", and the standard cluster one could be "group-du-worker-validator" - That's the main difference, which mcp pool are we watching; master or worker...

Reply:
But aside from this minor net, this looks great!

Reply:
Yeah, the only difference is the mcp. I like the idea. To make clarity of which one is for sno&3node and which one is for standard, I would keep the yaml file names and only change the PGT name.